### PR TITLE
fix(tests): more timeout and fix element .end() for OAuth tests

### PR DESCRIPTION
**Pull request by vladikoff**

### CODE REVIEW

File: tests/functional/oauth_webchannel.js [hunk 1]
```diff
@@ -257,6 +264,7 @@ define([
 
     'password reset, verify same browser': function () {
       var self = this;
+      self.timeout = 90 * 1000;
```
Comment (zaach) [inline tests/functional/oauth_webchannel.js]: Maybe we could make  `90 * 1000` a constant to DRY things up.

Comment (vladikoff) [inline tests/functional/oauth_webchannel.js]: Done.

**Comment by vladikoff** [comment]
@shane-tomlinson @rfk @zaach r?

`npm run test-functional-oauth`

and 


node_modules/.bin/intern-runner config=tests/intern_functional_oauth fxaAuthRoot=https://latest.dev.lcip.org/auth/v1 fxaContentRoot=https://latest.dev.lcip.org/ fxaEmailRoot=http://restmail.net fxaOauthApp=https://123done-latest.dev.lcip.org/ fxaIframeOauthApp=https://123done-latest.dev.lcip.org/iframe fxaUntrustedOauthApp=https://321done-latest.dev.lcip.org/ fxaProduction=true fxaToken=https://token.dev.lcip.org/1.0/sync/1.5

should PASS

**Comment by vladikoff** [comment]
I think I missed a few `.execute(listenForSyncCommands)`...

**Comment by vladikoff** [comment]
Fixes https://github.com/mozilla/fxa-content-server/issues/2407

**Comment by zaach** [comment]
:+1:

**Pull request merged**